### PR TITLE
MM-38533 Fixing themeing in markdown textboxes.

### PR DESCRIPTION
**Pull request by crspeller**
#### Summary
Simple CSS fix.

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]
Looks good -- but whitespace is off?

**Comment by crspeller** [comment]
@lieut-data Looks like a GitHub rendering issue.

File: webapp/src/components/backstage/styles.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const StyledTextarea = styled.textarea`
 
 export const StyledMarkdownTextbox = styled(MarkdownTextbox)`
     .custom-textarea {
-        background: white;
+		background-color: var(--center-channel-bg);
```
Comment (calebroseland) [inline webapp/src/components/backstage/styles.tsx:79]: Looks like tabs
```suggestion
       background-color: var(--center-channel-bg);
```

**Review by calebroseland — APPROVED** [review]

File: webapp/src/components/backstage/styles.tsx (hunk 1 above)
Comment (crspeller) [reply webapp/src/components/backstage/styles.tsx]: Strange. How did that happen...

**Pull request merged**